web/auth/guardian_serializer.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CodeCorps.GuardianSerializer do
+  @behaviour Guardian.Serializer
+
+  alias CodeCorps.Repo
+  alias CodeCorps.User
+
+  def for_token(user = %User{}), do: { :ok, "User:#{user.id}" }
+  def for_token(_), do: { :error, "Unknown resource type" }
+
+  def from_token("User:" <> id), do: { :ok, Repo.get(User, id) }
```

Comment:
I think we do need this, don't we? Regardless of whether we need all the user's information, we have to hit the database to see if that user exists.

This is from the README of Guardian itself.

Is there something I'm missing here?